src/contracts/middleware/example/HashThreshold.sol

```diff
@@ -0,0 +1,110 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+
+import "../../interfaces/IServiceManager.sol";
+import "./ECDSARegistry.sol";
+
+/**
+ * @title An EigenLayer middleware example service manager that slashes validators that sign a message that, when hashed 1000 types starts with less than 5 0s.
```

Comment:
please change 1000 hashes to something lighter-weight for testing. recommending 10x, can rename `kilohash` => `decahash`

Reply:
I recommend 69 hashes or septuagintahash

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/7/commits/68f80a0c541b8ba06d37d8840a96fe9e4beee6e1